proposal/nested-map-functions.md

```diff
@@ -0,0 +1,204 @@
+# Nested Map Functions: Draft 1.0
+
+*[(Issue)](https://github.com/sass/sass/issues/1739)*
+
+This proposal updates the built-in `sass:map` module to better support merging,
+setting, and getting items from nested maps.
+
+## Table of Contents
+
+* [Background](#background)
+* [Summary](#summary)
+* [Syntax](#syntax)
```

Comment:
Nit: We usually use the "Syntax" header to describe new syntax that literally has to be parsed in a different way than it was before. I'd call this section "Functions" to match https://github.com/sass/sass/blob/master/accepted/more-math-functions.md.